cobhan/Cargo.toml

```diff
@@ -2,8 +2,8 @@
 name = "cobhan"
 version = "0.1.0"
 edition = "2018"
-authors = ["Jeremiah Gowdy <jeremiah@gowdy.me>"]
-description = "Cobhan FFI Rust Library"
+authors = ["Jeremiah Gowdy <jeremiah@gowdy.me>", "Jeremiah Senkpiel <fishrock123@rocketmail.com>"]
+description = "Cobhan FFI is a system for enabling shared code to be written in Rust and consumed from all major languages/platforms in a safe and effective way, using easy helper functions to manage any unsafe data marshaling."
```

Comment:
Technically this can go either way, i.e. write something in Go and consume in Rust